src/v2/style-guide/index.md

```diff
@@ -1770,15 +1763,10 @@ button {
 
 ### Comunicação implícita entre componentes pai-filho <sup data-p="d">use com cuidado</sup>
 
-**Props and events should be preferred for parent-child component communication, instead of `this.$parent` or mutating props.**
-
-
 **Props e eventos devem ser usados preferencialmente para a comunicação do componente pai-filho, em vez de `this.$Parent`.**
```

Comment:
O código `this.$parent` está grafado incorretamente e falta o final da frase na tradução (propriedades de mutação).